utilslib/dr.py

```diff
@@ -91,10 +93,13 @@ def parse_key(key):
         tuble containing the fields in the key based on '/' seperator.
         """
         fields = key.split('/')
-        if len(fields) != 4:
+        if len(fields) == 4:
+            return fields[0], fields[1], fields[2], fields[3]
```

Comment:
I think we are expecting 5 fields? 
cluster
namespace
kind
version
name

Reply:
Yes we are but the s3 bucket has a mixture of both at present and so i'm just allowing both until we clear up s3.